[DEVCOMMS-474] Feature/standardize devpanelnaming (#1505)
mercadopago/devsite-docs: guides/online-payments/checkout-pro/previous-requirements.en.md

| Term | Description |
| --- | --- |
| `Preference`| It is the **information of the product or service that you want to offer.** Among the most important attributes of a preference are, namely, the description, the amount, and the items. When generating it, you get the URL to start the payment flow. |
| `Credentials` | Your credentials are the **unique keys we provide so you can configure your integrations.** There are two types: Public Key and Access Token. <br/><br/> **Public key is the application's public key** to know the payment methods and encrypt card details, for example. You must use it only for your integrations.<br/><br/> **Access Token is the application's private key** to generate payments. You must use it only for your integrations.<br/><br/>To find them, access the [Credentials]([FAKER][CREDENTIALS][URL]) section on your [Dashboard](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/resources/devpanel) and select the productive ones. |

> | `Initial Point (init_point)` | It is the **URL obtained at the time of generating the preference** and that starts the payment flow of the Checkout Pro. |

Reviewer: ```suggestion
| `Initial Point (init_point)` | It is the **URL obtained at the time of generating the preference**, which starts the payment flow of the Checkout Pro. |
```
